### PR TITLE
zh-translation:/docs/setup/additional-setup/cni/index.md

### PR DESCRIPTION
**Pull request by zhaosimon**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

**Comment by kebe7jun** [comment]
@zhaosimon 请先修复 lint 的检查。https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/9428/lint_istio.io/1377105356454367232

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 1]
```diff
@@ -107,29 +152,32 @@ Istio CNI 方案并非普遍应用的。一些平台，尤其是托管 Kubernete
...
 
-下列注入方式都是可以支持 Istio CNI 插件的：
+下列的 sidecar 注入方式都是可以支持 Istio CNI 插件的：
 
-1. [自动 sidecar 注入](/zh/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection)
-1. 使用 `istio-sidecar-injector` configmap 进行手动注入
-    1. 执行 [`istioctl kube-inject`](/zh/docs/reference/commands/istioctl/#istioctl-kube-inject) 直接使用 configmap：
+1. [sidecar 自动注入](/zh/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection)
+1. 使用 `istio-sidecar-injector` configmap 进行 sidecar 手动注入
+    1. 直接使用 configmap 执行 [`istioctl kube-inject`](/zh/docs/reference/commands/istioctl/#istioctl-kube-inject) ：
```
Comment (orangegzx) [inline content/zh/docs/setup/additional-setup/cni/index.md:173]:
```suggestion
    1. 直接使用 configmap 执行 [`istioctl kube-inject`](/zh/docs/reference/commands/istioctl/#istioctl-kube-inject)：
```

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 2]
```diff
@@ -179,11 +227,11 @@ Istio CNI 插件在容器运行时的进程空间内运行。因此 `kubelet` 
...
 使用 Istio CNI 时，`kubelet` 会通过以下步骤启动一个注入的 pod：
 
-1. Istio CNI 插件在 pod 内设置流量重定向到 Istio sidecar。
+1. Istio CNI 插件在 pod 内设置流量重定向到 Istio sidecar 代理。
 1. 等待所有的初始化容器成功执行完毕。
-1. Istio sidecar 跟随 pod 的其它容器一起启动。
+1. Istio sidecar 代理跟随 pod 的其它容器一起启动。
 
-初始化容器在 sidecar 启动之前执行，这会导致在它们执行期间会有流量丢失。
+初始化容器在 sidecar 代理启动之前执行，这会导致在它们执行期间会有流量丢失。
```
Comment (orangegzx) [inline content/zh/docs/setup/additional-setup/cni/index.md:234]:
```suggestion
初始化容器在 Sidecar 代理启动之前执行，这会导致在它们执行期间会有流量丢失。
```

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 3]
```diff
@@ -179,11 +227,11 @@ Istio CNI 插件在容器运行时的进程空间内运行。因此 `kubelet` 
 Istio CNI 插件可能会导致与应用 `initContainers` 的网络连通性。
 使用 Istio CNI 时，`kubelet` 会通过以下步骤启动一个注入的 pod：
 
-1. Istio CNI 插件在 pod 内设置流量重定向到 Istio sidecar。
+1. Istio CNI 插件在 pod 内设置流量重定向到 Istio sidecar 代理。
 1. 等待所有的初始化容器成功执行完毕。
-1. Istio sidecar 跟随 pod 的其它容器一起启动。
+1. Istio sidecar 代理跟随 pod 的其它容器一起启动。
```
Comment (orangegzx) [inline content/zh/docs/setup/additional-setup/cni/index.md:232]:
```suggestion
1. Istio Sidecar 代理跟随 pod 的其它容器一起启动。
```

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 4]
```diff
@@ -179,11 +227,11 @@ Istio CNI 插件在容器运行时的进程空间内运行。因此 `kubelet` 
 Istio CNI 插件可能会导致与应用 `initContainers` 的网络连通性。
 使用 Istio CNI 时，`kubelet` 会通过以下步骤启动一个注入的 pod：
 
-1. Istio CNI 插件在 pod 内设置流量重定向到 Istio sidecar。
+1. Istio CNI 插件在 pod 内设置流量重定向到 Istio sidecar 代理。
```
Comment (orangegzx) [inline content/zh/docs/setup/additional-setup/cni/index.md:230]:
```suggestion
1. Istio CNI 插件在 Pod 内设置流量重定向到 Istio Sidecar 代理。
```

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 5]
```diff
@@ -179,11 +227,11 @@ Istio CNI 插件在容器运行时的进程空间内运行。因此 `kubelet` 
 Istio CNI 插件可能会导致与应用 `initContainers` 的网络连通性。
 使用 Istio CNI 时，`kubelet` 会通过以下步骤启动一个注入的 pod：
```
Comment (orangegzx) [inline content/zh/docs/setup/additional-setup/cni/index.md:228]:
```suggestion
使用 Istio CNI 时，`kubelet` 会通过以下步骤启动一个注入的 Pod：
```

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 6]
```diff
@@ -107,29 +152,32 @@ Istio CNI 方案并非普遍应用的。一些平台，尤其是托管 Kubernete
...
+    $ istioctl install --set values.cni.cniBinDir=/home/kubernetes/bin \
+        --set components.cni.enabled=true \
+        --set components.cni.namespace=kube-system
     {{< /text >}}
 
+
 ## Sidecar 注入的兼容性{#sidecar-injection-compatibility}
 
-在 Helm 安装过程中需要用 `istio_cni.enabled=true` 生成 `istio-sidecar-injector` Configmap，使用这一配置对 Kubernetes Pod 进行 Istio 注入才能够使用 Istio CNI 插件。想了解更多 Sidecar 注入方面的详细内容请参考 [Istio Sidecar 注入文档](/zh/docs/setup/additional-setup/sidecar-injection/)。
+使用 Istio CNI 插件需要用 `istio_cni.enabled=true` 选项生成的 `istio-sidecar-injector` Configmap 对 Kubernetes Pod 的部署进行 Istio 注入。Sidecar 注入方面的详细内容请参考 [Istio Sidecar 注入文档](/zh/docs/setup/additional-setup/sidecar-injection/)。
```
Comment (Xunzhuo) [inline content/zh/docs/setup/additional-setup/cni/index.md:167]: Prefer ConfigMap

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 7]
```diff
@@ -107,29 +152,32 @@ Istio CNI 方案并非普遍应用的。一些平台，尤其是托管 Kubernete
...
 
-下列注入方式都是可以支持 Istio CNI 插件的：
+下列的 sidecar 注入方式都是可以支持 Istio CNI 插件的：
```
Comment (Xunzhuo) [inline content/zh/docs/setup/additional-setup/cni/index.md:169]: Prefer Sidecar

File: content/zh/docs/setup/additional-setup/cni/index.md (hunk 1 above)
Comment (Xunzhuo) [inline content/zh/docs/setup/additional-setup/cni/index.md:173]: Prefer ConfigMap

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 8]
```diff
@@ -107,29 +152,32 @@ Istio CNI 方案并非普遍应用的。一些平台，尤其是托管 Kubernete
...
+1. 使用 `istio-sidecar-injector` configmap 进行 sidecar 手动注入
+    1. 直接使用 configmap 执行 [`istioctl kube-inject`](/zh/docs/reference/commands/istioctl/#istioctl-kube-inject) ：
 
         {{< text bash >}}
         $ istioctl kube-inject -f deployment.yaml -o deployment-injected.yaml --injectConfigMapName istio-sidecar-injector
         $ kubectl apply -f deployment-injected.yaml
         {{< /text >}}
 
-    1. 用 configmap 创建文件，用于执行 `istioctl kube-inject`：
+    1. 用 configmap 创建的文件执行 `istioctl kube-inject`：
```
Comment (Xunzhuo) [inline content/zh/docs/setup/additional-setup/cni/index.md:180]: Prefer ConfigMap

**Review by Xunzhuo — COMMENTED** [review]
Generally LGTM，but need some small changes!

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 9]
```diff
@@ -32,14 +32,37 @@ Istio CNI 插件代替了 `istio-init` 容器所实现的功能。
...
+{{< /text >}}
+
+## 高级安装{#advanced-installation}
 
 1. 确认 Kubernetes 环境的 CNI 插件的 `--cni-bin-dir` 和 `--cni-conf-dir` 设置。
     任何非默认设置都需要参考[托管 Kubernetes 设置](#hosted-Kubernetes-settings)。
 
 1. 使用 `istioctl` 安装 Istio CNI 和 Istio。
-    参考 [Istio 安装](/zh/docs/setup/install/istioctl/)的说明，并设置 `--set components.cni.enabled=true` 和 `--set cni.components.cni.enabled=true` 选项。
+    参考 [Istio 安装](/zh/docs/setup/install/istioctl/)的说明并设置 `--set components.cni.enabled=true`选项。
```
Comment (Xunzhuo) [inline content/zh/docs/setup/additional-setup/cni/index.md:63]: 的说明并设置 `--set components.cni.enabled=true` 选项

File: content/zh/docs/setup/additional-setup/cni/index.md [hunk 10]
```diff
@@ -57,12 +80,21 @@ Istio CNI 插件代替了 `istio-init` 容器所实现的功能。
 | `cniConfFileName` | | | 不设置会自动查找 `cni-conf-dir` 目录的第一个文件（与 `kubelet` 一致）。主要用来测试 `install-cni` 的插件配置。如果设置了，`install-cni` 将会把插件配置注入到 `cni-conf-dir` 目录的该文件里面。 |
 | `psp_cluster_role` | | | 该值指的是一个 `ClusterRole` 并被用于在 `istio-cni` 的命名空间中创建一个 `RoleBinding`。当您使用 [Pod 安全策略](https://kubernetes.io/docs/concepts/policy/pod-security-policy)并且希望让 `istio-cni` 作为 `priviliged` Pods 运行时，这会非常有用。 |
 | `podAnnotations` | | `{}` | pod 级别自定义的附加注解。 |
+| `repair.enabled` | `boolean` | `true` | 启用或禁用 [CNI Race Condition](https://github.com/istio/istio/issues/14327) 的探测及修复功能。这会将 `istio-validation`  启动容器注入到每一个被注入的 pod 中用于检查 Istio CNI 是否正确初始化了 pod 的网络配置。这同时也在 CNI `DaemonSet` 中启用了一个新的容器用于监控 pods 并按照下列的数值打上标签或将其删除。|
```
Comment (Xunzhuo) [inline content/zh/docs/setup/additional-setup/cni/index.md:83]: pod to Pod

**Comment by windsonsea** [comment]
/close
since the author has no any response in 13 months.

**Pull request closed without merging**